src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -301,9 +304,9 @@ export class AccountDialog extends Modal {
 		this._noaccountViewContainer!.hidden = true;
 		if (Iterable.consume(this._providerViewsMap.values()).length > 0) {
 			const firstView = this._providerViewsMap.values().next().value;
-			if (firstView instanceof AccountPanel) {
-				firstView.setSelection([0]);
-				firstView.focus();
+			if (firstView && firstView.view instanceof AccountPanel) {
```

Comment:
this is wrong, firstView will never be AccountPanel:
![image](https://user-images.githubusercontent.com/13777222/111848858-69d06080-88c9-11eb-9d6e-683eba1e937a.png)

so correcting it